src/dom/dom.js

```diff
@@ -3283,14 +3285,14 @@ p5.Element.prototype.size = function (w, h) {
         this.elt.height = aH;
       }
 
-      this.width = this.elt.offsetWidth;
-      this.height = this.elt.offsetHeight;
+      originalWidth = aW;
```

Comment:
It looks like we don't set `this.width` or `this.height` any more here, was that on purpose?

Reply:
I looked into my approach again and tried changing it to:
`this.elt.width = aW;
 this.elt.height = aH;`

`this._pInst._setProperty('width', aW);
this._pInst._setProperty('height', aH);`

still doesn't work though, I'm kind off stuck now.

Reply:
Do you need to also set `this.width = aW` and `this.height = aH`?